scripts/ci/docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
     environment:
       - MYSQL_ALLOW_EMPTY_PASSWORD=true
       - MYSQL_ROOT_HOST=%
+      - MYSQL_DATABASE=airflow
```

Comment:
With this change we now don't need the 5-setup-mysql-env.sh (or what ever that file is called) I don't think.

Reply:
removed `5-setup-mysql-env.sh`